lib/tokens/index.js

```diff
@@ -2,42 +2,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var crypto = require('crypto')
-var inherits = require('util').inherits
+const crypto = require('crypto')
```

Comment:
Everywhere else, I limited my *e6ialization* only to the lines I was otherwise touching. But this module seemed small enough to bite the bullet and change en masse.